virtcontainers/experimental/experimental.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package experimental
+
+import (
+	"fmt"
+)
+
+// Feature to be experimental
+type Feature string
+
+var (
+	supportedFeatures = make(map[Feature]struct{})
+)
+
+// Register register a new experimental feature
+func Register(feature Feature) error {
```

Comment:
Where do you actually call this? Shouldnt DEFAULTEXPFEATURES be registered?

Reply:
We don't have an experimental feature so far, this is currently only a framework.
https://github.com/kata-containers/runtime/pull/883 will be the first experimental feature in my mind.

Reply:
Yes, but I was expecting to see a call to register the list of experimental features , though this list would be empty at this point.  Should this not be part of the framework @WeiZhang555 ?

Reply:
I prefer to leave it in this PR, or the framework will be kind of incomplete :)